### ISSUE TITLE
Travis npm autopublish

### ISSUE THREAD

**Issue by JackuB** [apiaryio/deckardcain#17]
Could you add [npm release to Travis file](http://docs.travis-ci.com/user/deployment/npm/)? I guess it should be done by original author.

**Comment by honzajavorek** [comment]
Good idea, probably also for https://github.com/apiaryio/generator-apiary-lib
